Formula/just.rb

```diff
@@ -0,0 +1,19 @@
+class Just < Formula
+  desc "🤖 Just a command runner"
+  homepage "https://github.com/casey/just"
+  url "https://github.com/casey/just.git",
```

Comment:
Does this not build if you use the archive tarball?

Reply:
That's what I tried at first, but with no success.

Reply:
My mistake was that I had the wrong tarball cached, so the build step would always fail.

```
==> Downloading https://github.com/casey/just/archive/v0.3.1.tar.gz
Already downloaded: /Users/ezku/Library/Caches/Homebrew/just-0.3.1.tar.gz
==> cargo build --release
Last 15 lines from /Users/ezku/Library/Logs/Homebrew/just/01.cargo:
2017-10-13 14:56:03 +0300

cargo
build
--release

error: could not find `Cargo.toml` in `/private/tmp/just-20171013-31183-1fb8xx3` or any parent directory
```

The dist release of just, with only the prebuilt binary, had somehow ended up in the cache under the same name as the github source archive tarball. Deleting the file worked.